apps/extension/src/routes/popup/approval/approve-deny.tsx

```diff
@@ -19,7 +19,7 @@ export const ApproveDeny = ({
   useEffect(startCountdown, [startCountdown]);
 
   return (
-    <div className='fixed inset-x-0 bottom-0 flex flex-row flex-wrap justify-center gap-4 bg-black p-4 shadow-lg'>
```

Comment:
Now, this div will only go up to where the approve/deny buttons are, so the scroll bar no longer goes behind them.